src/index.js

```diff
@@ -0,0 +1,32 @@
+import './style.css';
+import Methods from './utils.js';
+import Interactive from './interactive.js';
+
+const todoForm = document.getElementById('todo-form');
+const edittodoForm = document.getElementById('edit-todo-item');
+const todoFormInput = document.getElementById('todo-input');
+const editTodoFormInput = document.querySelector('.todo-edit-input');
+
+todoForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  Methods.addTodoTask(todoFormInput.value);
+  todoFormInput.value = '';
+});
+
+edittodoForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const id = Number(editTodoFormInput.getAttribute('id'));
```

Comment:
For JS Best Practices, It's better to avoid using objects for the primitive values (Numbers, Strings ...)
Example : 
 ```  const str = new String('name'); ``` , this is considered a bad practice, you can use instead:
```    const str = "name";```